### ISSUE TITLE
Add example of expect(response).to have_http_status(:status_code_name)

### ISSUE THREAD

**Issue by eliotsykes** [eliotsykes/rspec-rails-examples#10]
Example of using [`have_http_status`](https://relishapp.com/rspec/rspec-rails/docs/matchers/have-http-status-matcher) method

```ruby
# Rails `:symbolized` status codes at end of each status code page at http://httpstatus.es/
expect(response).to have_http_status(:success) # 200
expect(response).to have_http_status(:forbidden) # 403
```
Source: [RSpec Controller Specs cheatsheet](https://gist.github.com/eliotsykes/5b71277b0813fbc0df56#file-rspec_cheatsheet_controller_spec-rb-L34-L36)

**Comment by eliotsykes** [comment]
Consider adding `expect(page).to have_http_status(:success)` to a feature spec. rspec-rails supports this according to README.

**Comment by dideler** [comment]
Neat, I didn't know about `have_http_status` :bow: 

Some other options you can consider for controller and request specs to avoid magic numbers
- `assert_response :ok  # From Rails' Action Controller`
- `expect(response).to be_forbidden # Also from Rails' Action Controller`
- `it { should respond_with(:unauthorized) }  # From Shoulda Matchers`

**Comment by eliotsykes** [comment]
Thanks for the tips @dideler - especially like `should respond_with(...)` :+1:

**Comment by eliotsykes** [comment]
WIP: `have_http_status(...)` appearing in #96

**Comment by eliotsykes** [comment]
Closed by #96
